label_studio/users/models.py

```diff
@@ -41,6 +46,15 @@ def _create_user(self, email, password, **extra_fields):
 
         return user
 
+    def get_queryset(self):
+        qs = super().get_queryset()
+        if not self.disable_is_deleted_filter:
+            qs = qs.filter(is_deleted=False)
+        return qs
+
+    def with_deleted(self):
+        return super().get_queryset()
```

Comment:
Perhaps `with_soft_deleted`?

Reply:
On further reflection, I would actually support implementing `with_deleted` as a different manager on the model class instead of a method within this manager. This would unlock writing queries like `Users.with_deleted.<method>` instead of `Users.objects.with_deleted().<method>`